app/assets/stylesheets/views/_topics.scss

```diff
@@ -1,5 +1,12 @@
 .topics-page {
 
+  // Topic beta labels have a lot of text. Limiting the width of the label
+  // prevents the lines to wrap under the label.
+  .topic-beta-label .govuk-beta-label span {
+    width: 75%;
+    vertical-align: top;
+  }
```

Comment:
Cool, @alextea is going to try to make it multiline, as it's already broken in `govuk_frontend_toolkit`:

http://govuk-component-guide.herokuapp.com/component/beta_label

![screen shot 2015-05-28 at 15 10 18](https://cloud.githubusercontent.com/assets/233676/7862078/acf29bf6-054b-11e5-8970-36ca3b24a656.png)

Then we don't need this extra styling anymore.